bytecomp/dll.ml

```diff
@@ -174,10 +174,7 @@ let init_compile nostdlib =
 (* Initialization for linking in core (dynlink or toplevel) *)
 
 let init_toplevel dllpaths =
-  search_path :=
-    ld_library_path_contents() @
-    dllpaths @
-    ld_conf_contents();
+  search_path := dllpaths;
   opened_dlls :=
```

Comment:
Can you explain this change ?

Reply:
The search path is not changed.

The `dllpaths` passed to `init_toplevel` now contains the dll paths from `CAML_LD_LIBRARY_PATH` and from `ld.conf` as parsed by the runtime, rather than requiring `dll.ml` to parse these again.